x/feegrant/go.mod

```diff
@@ -0,0 +1,155 @@
+module github.com/cosmos/cosmos-sdk/x/feegrant
```

Comment:
From last SDK meeting discussion the result was that we go with vanity urls.

cosmossdk.io/x/feegrant in this case.